src/coreclr/vm/ceeload.cpp

```diff
@@ -6783,8 +6138,7 @@ idTypeSpec Module::LogInstantiatedType(TypeHandle typeHnd, ULONG flagNum)
     {
         NOTHROW;
         GC_NOTRIGGER;
-        MODE_ANY;
```

Comment:
You should be able to delete LogInstantiatedType and LogInstantiatedMethod. As well as TokenDefinitionHelper and Module::EncodeModuleHelper.